app/models/like.rb

```diff
@@ -6,6 +6,6 @@ class Like < ApplicationRecord
   private
 
   def increment_likes_counter
-    post.increment!(:likes_counter)
+    post.increment!(:LikesCounter)
   end
```

Comment:
- [ ]  Good job with test coverage 💪, but since we are aiming for 100% test coverage, and it is also a [requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md)for this project, "Add unit specs for all of your models' methods and validations", please add a unit test for `increment_likes_counter` method.